app/lib/frontend/search_service.dart

```diff
@@ -210,12 +218,16 @@ class SearchQuery {
     this.offset: 0,
     this.limit: 10,
     this.type,
+    this.packagePrefix,
     this.bias,
   });
 
   /// Returns the query text to use in CSE.
   String buildCseQueryText() {
     String queryText = text;
+    if (packagePrefix != null) {
+      queryText += ' $packagePrefix';
```

Comment:
we remove the package:[whatever] pattern from the original query string, and in case of CSE, adding it to the query may actually yield some good (although far for correct) results.